client/sass/_depth-gauges.scss

```diff
@@ -204,19 +205,34 @@ $left-offset: -3.65em;
   display: flex;
   overflow-x: auto;
   flex-wrap: nowrap;
+}
```

Comment:
It would be best to stick to the SCSS formatting of having the child element styling nested inside of the parent for more direct specificity.